index.css

```diff
@@ -0,0 +1,10 @@
+main {
+  color: white;
+  background-color: gray;
+}
+
+h1 {
+  font-family: sans-serif;
+  font-weight: 80px;
+  color: orange;
+}
```

Comment:
- [ ] Try adding an extra font family, as some browsers may not accept the first
- [ ] Using hex codes as colours are preferable by the browser to original colour names, try using then